apps/falcon_app.py

```diff
@@ -31,10 +32,49 @@ def on_post(self, req, resp):
         resp.media = {"status": "ok", "md5": digest}
 
 
+def thread_function(user_input):
+    """This is intented to test Contrast specific functionality!"""
+
+    print("starting background thread")
+
+    # We're testing that a child thread still has a request context
+    # even after the parent thread exited, so waiting here so the parent
+    # thread has time to exit.
+    time.sleep(2)
+
+    import contrast
```

Comment:
Probably no one else will use this view, but do we still need to handle `ImportError` here?

Reply:
Yeah I can do that so it doesn't break.